docs/source/search-runs.rst

```diff
@@ -156,6 +156,23 @@ You can search using the following run attributes contained in :py:class:`mlflow
   attributes.run_id = 'a1b2c3d4'
   attributes.run_id IN ('a1b2c3d4', 'e5f6g7h8')
 
+Datasets
+~~~~~~~~~~~~~~
+
```

Comment:
If it's not too much trouble, can we also add a datasets search example to the UI tooltip that pops up next to the runs search bar on the MLflow Experiment Page UI?